avocado/core/remote/runner.py

```diff
@@ -180,27 +142,16 @@ def run_test(self, urls, timeout):
         :return: a dictionary with test results.
         """
         extra_params = []
-        mux_files = [os.path.join(self.remote_test_dir, mux_file)
-                     for mux_file in getattr(self.job.args,
-                                             'mux_yaml') or []]
+        mux_files = [mux_file for mux_file in getattr(self.job.args,
+                                                      'mux_yaml') or []]
         if mux_files:
             extra_params.append("-m %s" % " ".join(mux_files))
 
         if getattr(self.job.args, "dry_run", False):
             extra_params.append("--dry-run")
         urls_str = " ".join(urls)
-        avocado_check_urls_cmd = ('cd %s; avocado list %s '
```

Comment:
OK, we don't need to discover them as we'll get the error message from the main program instead, I like this change...